BUILD.bazel

```diff
@@ -14,7 +14,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+load("@buildifier_prebuilt//:rules.bzl", "buildifier")
```

Comment:
Noob question. Assuming I'm building without bzlmod and use the `load_repositories.bzl` in my WORKSPACE file, then this will break, right?

If yes, can you please adjust the other places where `com_github_bazelbuild_buildtools` was used before?

Reply:
adjusted [here]()